flink-core/src/main/java/org/apache/flink/api/common/restartstrategy/RestartStrategies.java

```diff
@@ -54,6 +55,18 @@ public static RestartStrategyConfiguration fixedDelayRestart(
 		return new FixedDelayRestartStrategyConfiguration(restartAttempts, delayBetweenAttempts);
 	}
 
+	/**
+	 * Generates a FailureRateRestartStrategyConfiguration.
+	 *
+	 * @param maxFailureRate Maximum number of restarts in given time unit {@code failureRateUnit} before failing a job
+	 * @param failureRateUnit Time unit for measuring failure rate
+	 * @param delayBetweenRestartAttempts Delay in-between restart attempts
+	 */
+	public static FailureRateRestartStrategyConfiguration failureRateRestart(
+			int maxFailureRate, TimeUnit failureRateUnit, long delayBetweenRestartAttempts) {
```

Comment:
Maybe we could change the parameter list to `failureRateRestart(int rate, long interval, TimeUnit intervalUnit, long delay, TimeUnit delayUnit)`